.eslintrc

```diff
@@ -17,6 +17,7 @@ globals:
     require: true
     module: true
     expect: true
+    assert: true
```

Comment:
I don't think we need `assert` as the new place it was used is probably better served with a first-class `sinon-chai` assert :)

Reply:
done.